src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -55,14 +57,23 @@
     @Nullable
     private EventTypeOptions options;
 
-    public EventType() {}
+    private Set<String> writeScopes;
+
+    private Set<String> readScopes;
+
+    public EventType() {
+        this.validationStrategies = Lists.newArrayList();
+        this.enrichmentStrategies = Lists.newArrayList();
+        this.partitionStrategy = PartitionStrategy.RANDOM_STRATEGY;
+        this.writeScopes = Collections.emptySet();
+        this.readScopes = Collections.emptySet();
+    }
 
     public EventType(final String name, final String topic, final String owningApplication, final EventCategory category,
                      final List<ValidationStrategyConfiguration> validationStrategies,
                      final List<EnrichmentStrategyDescriptor> enrichmentStrategies, final String partitionStrategy,
                      final List<String> partitionKeyFields, final EventTypeSchema schema, final EventTypeStatistics defaultStatistic,
-                     final EventTypeOptions options)
-    {
+                     final EventTypeOptions options, final Set<String> writeScopes, final Set<String> readScopes) {
```

Comment:
I agree @rcillo but it is used only in one place for test purposes because of that fact I have not touched it.